qutebrowser/config/configdata.py

```diff
@@ -227,6 +227,16 @@ def data(readonly=False):
              "How to open links in an existing instance if a new one is "
              "launched."),
 
+            ('new-instance-open-target.window',
+             SettingValue(typ.String(
+                 valid_values=typ.ValidValues(
+                     ('last-opened', "Open new tabs in the last"
```

Comment:
You're missing a space before the `"` here and below, but I'll fix it up when merging.